### ISSUE TITLE
Noisy drag events

### ISSUE THREAD

**Issue by ckritzinger** [cameronhimself/vue-drag-drop#19]
While dragging, the Drag component seems to emit a drag event every couple of milliseconds.  This swamps the event view in my DevTools (to the point of hanging up my DevTools).

Would be really handy if there was a way of suppressing unwanted events.

**Comment by cameronhimself** [comment]
Having given this some more thought, I realized I closed this too hastily. Will look into it.

**Comment by ckritzinger** [comment]
Yay :) I had a quick look at the source and I think I can see a reasonably elegant way of doing it.  Happy to have a bash and make you a PR if you like?

**Comment by cameronhimself** [comment]
By all means. I already took a crack adding a milliseconds `debounce` prop, and using the [`debounce`](https://www.npmjs.com/package/debounce) package to slow the event emission, but it was behaving strangely. Maybe you'll have more luck.

Also, you'll want to PR into the `release/1.0.0` branch. It should be easier to work in, since the code is simplified.

**Comment by pdcmoreira** [comment]
Is this abandoned?

**Comment by AMoreaux** [comment]
I have the same issue 😞
@pdcmoreira Did you find a temporary solution?

**Comment by pdcmoreira** [comment]
@AMoreaux no, it's just a bit annoying when debugging, but everything else is fine.
